docs/adr-dependencies.md

```diff
@@ -0,0 +1,86 @@
+# usage of Dependencies, peerDependencies, devDependencies in package.json
+
+## Context
+
+Dependencies are specified using the package.json of libraries.
+We have more and more libraries written and used by projects.
+
+We use different distribution format for our libraries:
+
+- UMD for some (react-components, react-cmf, etc ...)
+- mix of transpiled js and sass files which need shared config to make it work for others
+
+## Problems
+
+We see commits with mix of peerDependencies and dependencies without real guidelines.
+Each change in peerDependencies lead to a major release.
```

Comment:
breaking change means project has to do a change. If a library change a peerDependencies it means user (project) have to do change. Even minor.

Reply:
> If a library change a peerDependencies it means user (project) have to do change.
If a library's peerDep implements a bugfix, the user of that library doesn't have to change anything. You do get a warning, but there's no actual *need* for the host project to change at this point. 
That's the whole benefit of semver too: if any minor change is as pressing as a major one, then they are all major.

It would be very paralysing to keep enforcing this IMO.

Reply:
https://github.com/semver/semver/issues/502 this has been discussed here